_includes/v2.1/app/common-steps-secure.md

```diff
@@ -0,0 +1,84 @@
+## Step 2. Start a cluster
+
+Before we can run code, we need to start a cluster.  Follow the steps below to start a secure local cluster with 3 nodes.
+
+1. Create a CA (certificate authority) that will be used to sign all of your other certificates.
+
+    {% include copy-clipboard.html %}
+    ~~~ sh
```

Comment:
Normally we ask users to create the certs directory and then create the certificates within that directory. I am curious as to why did we change the directory to /tmp/certs/ and if we should change it elsewhere as well.

Reply:
Also, copy-pasting and running the command gives an error because the directory doesn't exist. I know it's a simple error that the user can fix, but just FYI

Reply:
`--certs-dir=/tmp/certs--certs-dir=/tmp/certs` should be `--certs-dir=/tmp/certs`